java/code/src/com/redhat/rhn/domain/recurringactions/OrgRecurringAction.java

```diff
@@ -80,7 +80,8 @@ public OrgRecurringAction(boolean testMode, boolean active, Org org, User creato
      */
     @Override
     public boolean canAccess(User user) {
-        return user.hasRole(RoleFactory.ORG_ADMIN) && user.getOrg().equals(getOrg());
+        return (user.hasRole(RoleFactory.ORG_ADMIN) && user.getOrg().equals(getOrg())) ||
```

Comment:
I just want to add: despite my comment in the previous commit, this commit makes sense as `SAT_ADMIN` really has rights to access/edit the action. This fix is very important for the XMLRPC API, btw.